File: 0682/sesión 13 - MasterPage+CRUD/con CRUD/SoftInv/SoftInvPersistance/DAOImpl/Util/Columna.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftInv.DAOImpl.Util
{
    public class Columna
    {
        private string nombre;
        private bool? esLlavePrimaria;
        private bool? esAutoGenerado;

        public Columna()
        {
            this.Nombre = null;
            this.EsLlavePrimaria = null;
            this.EsAutoGenerado = null;
        }

        public Columna(string nombre, bool esLlavePrimaria, bool esAutoGenerado)
        {
            this.Nombre = nombre;
            this.EsLlavePrimaria = esLlavePrimaria;
            this.EsAutoGenerado = esAutoGenerado;
        }

        public string Nombre { get => nombre; set => nombre = value; }
        public bool? EsLlavePrimaria { get => esLlavePrimaria; set => esLlavePrimaria = value; }
        public bool? EsAutoGenerado { get => esAutoGenerado; set => esAutoGenerado = value; }
    }
}

File: 0682/sesión 20 - RESTful (1)/ServidorRESTFul/SoftInvPersistance/src/main/java/pe/edu/pucp/softinv/daoImp/util/Columna.java
package pe.edu.pucp.softinv.daoImp.util;

//esta clase representa la clase que incorporará los metadatos a la solución
public class Columna {
    private String nombre;
    private Boolean esLlavePrimaria;
    private Boolean esAutoGenerado;

    public Columna(String nombre, Boolean esLlavePrimaria, Boolean esAutoGenerado) {
        this.nombre = nombre;
        this.esLlavePrimaria = esLlavePrimaria;
        this.esAutoGenerado = esAutoGenerado;
    }
        
    /**
     * @return the nombre
     */
    public String getNombre() {
        return nombre;
    }

    /**
     * @param nombre the nombre to set
     */
    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    /**
     * @return the esLlavePrimaria
     */
    public Boolean getEsLlavePrimaria() {
        return esLlavePrimaria;
    }

    /**
     * @param esLlavePrimaria the esLlavePrimaria to set
     */
    public void setEsLlavePrimaria(Boolean esLlavePrimaria) {
        this.esLlavePrimaria = esLlavePrimaria;
    }

    /**
     * @return the esAutoGenerado
     */
    public Boolean getEsAutoGenerado() {
        return esAutoGenerado;
    }

    /**
     * @param esAutoGenerado the esAutoGenerado to set
     */
    public void setEsAutoGenerado(Boolean esAutoGenerado) {
        this.esAutoGenerado = esAutoGenerado;
    }
    
    
}
